dev/com.ibm.ws.security.token.ltpa/src/com/ibm/ws/security/token/ltpa/LTPAValidationKeysInfo.java

```diff
@@ -61,14 +61,14 @@ public LTPAPublicKey getLTPAPublicKey() {
         return ltpaPublicKey;
     }
 
-    public boolean isNotUseAfterDate() {
-        if (notUseAfterDateOdt == null)
+    public boolean isValidUntilDate() {
```

Comment:
same comment here. except the logic is flipped where it returns true if it is before not after
```suggestion
    public boolean isBeforeValidUntilDate() {
```